src/components/GridLayout.js

```diff
@@ -60,184 +55,23 @@ class GridLayout extends Component {
-                        }}
-                      />
-                    </div>
-                    <div>
-                      <h5>Time (e.g. "09:15 AM")</h5>
-                      <input
-                        type="time"
-                        name="time"
-                        value={this.props.time}
-                        onChange={event => {
-                          this.props.changeTime(event.target.value);
-                        }}
-                      />
-                    </div>
-                  </div>
-                  <br />
-                  <button
-                    className={styles.OKButton}
-                    onClick={() => this.setState({ settingsMenu: false })}
-                  >
-                    OK
-                  </button>
-                </div>
-              ) : null}
-              {settingsMenuId === 1 ? (
-                <div style={{ padding: 20 }}>
-                  <h4 style={{ padding: 0, margin: 0 }}>Map settings</h4>
-                  <hr />
-                  <div className={styles.MapSettings}>
-                    <p>
-                      There are {MAP_BACKGROUNDS
-                        ? MAP_BACKGROUNDS.length
-                        : 0}{" "}
-                      available map background(s):
-                      {MAP_BACKGROUNDS[0].description} and{" "}
-                      {MAP_BACKGROUNDS[1].description}.
-                    </p>
-                    <p>
-                      Currently selected:&nbsp;
-                      <strong>
-                        {this.props.currentMapBackground.description}
-                      </strong>.
-                    </p>
-                    <button onClick={this.toggleMapBackground.bind(this)}>
-                      Switch
-                    </button>
-                  </div>
-                  <br />
-                  <button
-                    className={styles.OKButton}
-                    onClick={() => this.setState({ settingsMenu: false })}
-                  >
-                    OK
-                  </button>
-                </div>
-              ) : null}
-
-              {settingsMenuId === 2 ? (
-                <div style={{ padding: 20 }}>
-                  <h4 style={{ padding: 0, margin: 0 }}>Contact info</h4>
-                  <hr />
-                  <p>
-                    For software issues with the FloodSmart Parramatta System
-                    please contact Nelen & Schuurmans on {nensMail()}. For any
-                    other issues, or suggestions for improvements to the
-                    Parramatta Floodsmart System system, please contact Chris
-                    Gooch on tel.&nbsp;
-                    {chrisTel()} or email {chrisMail()}
-                  </p>
-                </div>
-              ) : null}
-            </main>
-          </div>
-        </DocumentTitle>
+        <SettingsMenu
```

Comment:
SettingsMenu component is called instead of hardcoded inside this component. Also CloseSettingsMenu function is passed as prop